README.rst

```diff
@@ -40,7 +40,7 @@ Example usage with lxml:
 .. code-block:: python
 
     from lxml.html import fromstring
-    from clean_html import clean_node, cleaned_node_to_html
+    from clear_html import clean_node, cleaned_node_to_html
```

Comment:
Should we change the API to call the functions clear.. as well?

Reply:
I don't know. Initially I decided to keep it because the old names sound better to me, but maybe it's better to have them match the module name. @Gallaecio what do you think?